modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TaskHeaderActions.tsx

```diff
@@ -57,14 +58,11 @@ const TaskHeaderActions = () => {
 	);
 	const [userIds, setUsersId] = useState<number[]>([]);
 	const {modal} = useFormModal({
-		onSave: (newUserIds: number[]) =>
-			testrayTaskImpl
-				.assignTo(testrayTask, newUserIds)
-				.then((response) => {
-					mutateTask(response);
-
-					revalidateTaskUser();
-				}),
+		onSave: async (newUserIds: number[]) => {
+			await testrayTaskUsersImpl.assign(testrayTask.id, newUserIds);
+
+			revalidateTaskUser();
+		},
```

Comment:
This is not wrong, but we discussed about the possibility of newUserIds be equal to 0 and remove all the assignees.

It should be at least 1 (throw an alert, the same used in TestflowForm.tsx)